sdk/servicebus/azure-servicebus/README.md

```diff
@@ -80,6 +80,16 @@ Once you've initialized a `ServiceBusClient`, you can interact with the primary
 
 For more information about these resources, see [What is Azure Service Bus?][service_bus_overview].
 
+To interact with these resources, one should be familiar with the following SDK concepts:
+
+* [ServiceBusClient](./azure/servicebus/_servicebus_client.py): This is the object a user should first initialize to connect to a Service Bus Namespace.  To interact with a queue topic or subscription, one would spawn a sender or receiver off of this client.
+
+* [Sender](./azure/servicebus/_servicebus_sender.py): To send messages to a Queue or Topic, one would use the corrosponding `get_queue_sender` or `get_topic_sender` method off of a `ServiceBusClient` instance as seen [here](./samples/sync_samples/send_queue.py)
```

Comment:
nit: missing period at the end.

Reply:
`get_topic_sender` isn't available for p1,
we should either consider removing this or mention that it will coming in the next release.